src/averageAges.js

```diff
@@ -37,7 +47,19 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const selection = people.filter(woman => {
+    const check = woman.sex === 'f' && withChildren
```

Comment:
does this make sense to you -checking whether a woman's sex is feminine? 😄

Reply:
Well, you never know 😁 better make sure
👌fixed